index.html

```diff
@@ -224,6 +224,36 @@ <h3 class="connect-text">LET'S CONNECT</h3>
       </div>
     </div>
   </section>
+
+  <section class="contact-form">
+    <div class="contact-form-holder">
+      <h2 class="contact-title">Contact me</h2>
+
+      <p class="p-pargraph">
+        If you have an application you are interested in developing, a feature
+        that you need built or a project that needs coding. I’d love to help
+        with it
+      </p>
+
+      <form action="https://formspree.io/f/xqkwonvv" class="form-contact" method="post">
+        <ul class="ul-form">
+          <li class="li-form">
+            <input type="text" placeholder="Emmanue Hagan" name="name" id="name_form" maxlength="30" required />
+          </li>
+          <li class="li-form">
+            <input type="email" placeholder="haganemmanuel@gmail.com" name="email" id="email_form" required />
+          </li>
+          <li class="li-form">
+            <textarea name="message" placeholder="Write your message here" id="message" cols="30" rows="10"
+              maxlength="500" required></textarea>
```

Comment:
- [ ] please try to add some padding to the `<input>` tags in order they will be as ones in the Figma design 

- [ ] Yours 
![image](https://user-images.githubusercontent.com/61790587/166972892-2ebf6058-dd2a-4906-99bf-061786be9467.png)

- [ ] Figma design 
![image](https://user-images.githubusercontent.com/61790587/166973010-5061b239-e6d1-46c1-8a0c-1e62cde2fe3e.png)